renderer/src/components/FilAddressForm.tsx

```diff
@@ -92,7 +108,21 @@ const FilAddressForm: FC<FilAddressFormProps> = ({
         <p
           className={`
             absolute text-body-2xs text-white mt-3 ease-[cubic-bezier(0.85,0,0.15,1)] duration-500
-            ${(internalEditMode && addressIsValid) ? 'visible' : 'invisible opacity-0'}
+            ${(internalEditMode && addressIsValid && inputAddr.startsWith('f1'))
+              ? 'visible'
+              : 'invisible opacity-0'
+            }
+          `}
+        >
+          Warning: f1 addresses require extra gas for the FilForwarder contract
```

Comment:
I am proposing to clarify this warning - it's not the f1 address but the transaction sending money that requires extra gas fee.
```suggestion
          Warning: Sending rewards to f1 addresses incurs a higher gas fee. Consider using an f4 address instead.
```

@patrickwoodhead Could you suggest a better way how to convey this information? Leaning on you as the only native speaker in our team 😄

Reply:
What about "Sending rewards to an f1 address incurs a higher gas fee than sending to an f4 address because it invokes the FilForwarder contract. Consider using an f4 address instead."

Reply:
<img width="549" alt="Screenshot 2023-10-19 at 12 44 35" src="https://github.com/filecoin-station/desktop/assets/10247/ecb69078-8d6a-4685-9302-7fdc9d71fe28">

We're almost running out of space here. It doesn't yet cause layout issues, it isn't pretty, but it works

Reply:
LGTM.

I would add a bit of space between the warning and the header "TOTAL FIL". Or perhaps wrap the warning in a border box (`border: solid 1px`). Not a big deal, I don't want to bikeshed.

Reply:
I assume we'll have to revisit this anyway, I'd rather get this out quicker

Reply:
And gain operational experience